core/sbt-cloudflow/src/main/scala/cloudflow/sbt/CommonSettingsAndTasksPlugin.scala

```diff
@@ -64,8 +64,16 @@ object CommonSettingsAndTasksPlugin extends AutoPlugin {
           }.value,
       publishArtifact in (Compile, packageDoc) := false,
       publishArtifact in (Compile, packageSrc) := false,
-      libraryDependencies += "com.twitter"     %% "bijection-avro" % "0.9.7",
-      libraryDependencies += "org.apache.avro" % "avro"            % "1.8.2",
+      // Add Avro libraries
+      libraryDependencies ++= Seq(
+            "com.twitter"     %% "bijection-avro" % "0.9.7",
+            "org.apache.avro" % "avro"            % "1.8.2"
+          ),
+      // Add protoc libraries
+      libraryDependencies ++= Seq(
+            "io.grpc"              % "grpc-netty"            % scalapb.compiler.Version.grpcJavaVersion,
```

Comment:
Can you use shaded grpc netty instead @blublinsky? https://github.com/akka/akka-grpc/issues/1 https://github.com/grpc/grpc-java/pull/2485

Reply:
Done